contextual_orchestrator/orchestrator.py

```
@@ -1542,20 +1550,25 @@ class _StateStore:
     """Minimal write-through sqlite persistence for orchestrator runtime state.
 
     ponytail: one generic table, no ORM. Keyed kinds (workflow_run, evaluation_run)
-    upsert by key; stream kinds (analytics, audit) append. Stream rows grow unbounded
-    on disk while the in-memory deques stay capped — add pruning if db size matters.
+    upsert by key; stream kinds append. Durable audit streams use the same bounded
+    retention as their in-memory deques so request traffic cannot grow the DB forever.
     Runtime values (kind, key, payload, limit) are always bound through SQLite
     placeholders so persisted prompts and identifiers cannot become SQL syntax.
     """
 
     _KEYED = {"workflow_run", "evaluation_run"}
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)